src/python/pants/core/util_rules/environments.py

```diff
@@ -426,7 +425,25 @@ async def determine_local_environment(
 
             To fix, either adjust the `{CompatiblePlatformsField.alias}` field from those
             targets so that only one includes the value `{platform.value}`, or change
-            `[environments-preview].names` so that it does not define some of those targets.
+            `[environments-preview].names` so that it does not define some of those targets. It is
+            often useful to still keep the same `local_environment` target definitions in BUILD
```

Comment:
```suggestion
            `[environments-preview].names` so that it does not define some of those targets.
            
            It is often useful to still keep the same `local_environment` target definitions in BUILD
```

IMO. The previous sentence stands alone as a complete explanation: the rest of this expands on it in more detail.